docs/sources/next/javascript-api/jslib/http-instrumentation-pyroscope/_index.md

```diff
@@ -0,0 +1,53 @@
+---
+title: "http instrument pyroscope"
+descriptiontion: "k6 pyroscope instrumentation API"
+weight: 04
+weight: 04
+---
+
+# Pyroscope
+
+
+With this jslib, you can _instrument_ HTTP requests so that Grafana Cloud Profiles can match profiles to Grafana Cloud K6 tests.
+
+## About baggage header
+
+A _barrage header_ is a standardized HTTP header to propagate distributed context. The [W3C specification](https://www.w3.org/TR/baggage/) goes into more details on the specifics, but like many other headers is a list of key, value  pairs. 
+
+This module by default adds a 3 key value pairs, scenario name, the name of the request (url if not set) and the value of `__ENV.K6_CLOUDRUN_TEST_RUN_ID` which is automatically in grafana k6 cloud.
```

Comment:
I'm not sure if you agree, but perhaps it would make sense to add a sentence or two about how the integration works on the Pyroscope side. WDYT? Maybe just mentioning that these key-value pairs will also be attached to the profiles and that the user can query them in the Pyroscope backend?

Reply:
Good edit and suggestion.